File: cases/taint/javascript/model-sanitizer-kill-positive/ModelSanitizerKillPositive.js
function dfb_source() { // DFB-SOURCE: model-sanitizer-kill-input
  return "tainted";
}

function dfb_sink(value) {} // DFB-SINK: model-sanitizer-kill-sink

const Clean = {
  scrub: function scrub(value) {
    return value;
  }
};

function run() {
  dfb_sink(dfb_source());
}

File: cases/taint/swift/model-sanitizer-kill-positive/ModelSanitizerKillPositive.swift
enum Clean {
    static func scrub(_ value: String) -> String { value } // DFB-MODEL: module=DataFlowBenchTaintSwift; identity=DataFlowBenchTaintSwift.Clean.scrub(_ value: Swift.String) -> Swift.String; role=sanitizer; binding=in:0
}

func dfb_source() -> String { "tainted" }
func dfb_sink(_ value: String) { print(value) }

func run() {
    let input = dfb_source() // DFB-SOURCE: model-sanitizer-kill-input
    dfb_sink(input) // DFB-SINK: model-sanitizer-kill-sink
}
